.github/workflows/clang.yaml

```diff
@@ -110,7 +110,7 @@ jobs:
           if [ -s testplan.json ]; then
             echo "::set-output name=report_needed::1";
             # Full twister but with options based on changes
-            ./scripts/twister --force-color --inline-logs -M -N -v --load-tests testplan.json --retry-failed 2
+            ./scripts/twister --force-color --inline-logs -M all-tests -N -v --load-tests testplan.json --retry-failed 2
```

Comment:
why do we need this change?

```suggestion
            ./scripts/twister --force-color --inline-logs -M -N -v --load-tests testplan.json --retry-failed 2
```

Reply:
> why do we need this change?

Just i want to CI check twister with this option after updating

Reply:
@stephanosio you mean that CI should check twister with default option` -M`

Reply:
yes, please leave that alone please, we will consider enabling this in CI later.